solr/core/src/java/org/apache/solr/handler/ReplicationHandler.java

```diff
@@ -1231,6 +1231,9 @@ private void setupPolling(String intervalStr) {
   @Override
   @SuppressWarnings({"resource"})
   public void inform(SolrCore core) {
+
+    //TODO core.getCoreContainer().getAllowListUrlChecker()
```

Comment:
I see you managed to get ahold of the checker from IndexFetcher so maybe it's not needed here in ReplicationHandler?

Reply:
Yes. This TODO was a remaining of some investigation. I'l just remove it.